packages/integrations/mdx/README.md

```diff
@@ -334,18 +337,17 @@ export const components = { blockquote: Blockquote };
 
 Now, writing the standard Markdown blockquote syntax (`>`) will use your custom `<Blockquote />` component instead. No need to use a component in Markdown, or write a remark/rehype plugin! Visit the [MDX website](https://mdxjs.com/table-of-components/) for a full list of HTML elements that can be overwritten as custom components.
 
-
 #### Custom components with imported `mdx`
 
-When rendering imported MDX content, custom components can also be passed via the `components` prop:
+When rendering imported MDX content, custom components must be passed via the `components` prop. If the imported MDX exports custom components, you can pass them as props. While doing that, you could also override some mappings:
```

Comment:
```suggestion
When rendering imported MDX content, custom components can be passed via the `components` prop. They can also be used to override mappings.

You can also import any exported components from an MDX file along with `Content`. Pass these in to your `<Content />` component using the same `components` prop.
```
The problem with saying "must" here is that it can be interpreted as you MUST pass some custom components when you render MDX. So, it seems more correct to say that you "can" and I would leave that part as is. The sentence reads to me like, "When you're rendering MDX, you can additonally choose at that time to do some extra stuff..." and I believe that is the intention?

(If the idea is that IF you want to pass components, you MUST do it this way, then "must" is not necessary here. I think showing that it is done this way achieves that goal of saying "this is how it must be done." )

The second sentence is not quite the language we'd use in the docs. I've taken a shot at rewriting what I think you're describing, and attempting to separate out the two ideas that the code sample is demonstrating. I'll ask @bholmesdev to take the final edits getting the nuance just right, and to make sure he likes this example. (I don't *think* it's too much for one code sample to add this extra bit in, if the explanation above it is clear enough, but I'm leaving that to Ben's call.)

Reply:
Hi Sarah, thanks for your review: You have a very good eye for consistent wording. Also your comment, to split the additional info out is very good; I leave it to @bholmesdev, whether he wants it in or not, I think it can be left out without loosing much.

I read the sentence with minimal change (your proposal, just removing the 'also') a few times before and now, and it makes the docs correct, but I am not sure if it makes the difference in behavior between the section above and this clear enough:

- using mdx as page: exported components work out of the box
- imported mdx: exported components do not work, but components MUST be passed in as props if you want to have custom components

At least for @bholmesdev and @aFuzzyBear it was also a surprise to see it not working in the reproduction scenario.